portfolio/src/main/webapp/script.js

```diff
@@ -12,19 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 /**
  * Adds a random fact to the page.
  */
 function addRandomFact() {
-  const facts =
-      ['I am the youngest in my family by 12.5 years.', 'I have 4 nephews.', 'I have been to every continent except South America and Antarctica.'];
 
-  // Pick a random fact.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
+  // Request for a random fact from server 
+  console.log("Fetching quote from server...");
+  const factPromise = fetch('/random-fact').then((res) => res.text());
+
 
   // Add it to the page.
```

Comment:
Single blank lines are encouraged for readability, but multiple blank lines are usually not. See https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace